src/components/PageHeading.js

```diff
@@ -1,22 +1,63 @@
 import { Component } from 'react';
 import { div, hh, img, h2, span } from 'react-hyperscript-helpers';
-import './PageHeading.css';
 
 export const PageHeading = hh(class PageHeading extends Component {
 
   render() {
 
-    return div({ id: this.props.id + "_heading", className: "page-heading"}, [
+    const HEADING = {
+      width: '100%',
+      margin: '20px 0 10px 0',
+      position: 'relative'
+    };
+
+    const ICON = {
+      position: 'absolute',
+      top: '0',
+      left: '0',
+      height: '50px'
+    };
+
+    const DESCRIPTION = {
+      color: '#000000',
+      height: '25px',
+      fontWeight: '400',
+      fontSize: '19px'
+    };
+
+    const TITLE = {
+      margin: '7px 0 5px 0',
+      lineBreak: 'auto'
+    };
+
+    const MEDIUM = {
+      marginLeft: '55px'
+    };
+
+    const LARGE = {
+      marginLeft: '70px'
+    };
+
+    const NONE = {
+      marginLeft: '0'
+    };
+
+
+    return div({ id: this.props.id + "_heading", style: HEADING}, [
       img({
         id: this.props.id + "_icon",
         isRendered: this.props.imgSrc !== undefined,
         src: this.props.imgSrc,
         alt: this.props.title,
-        className: "page-heading-icon"
+        style: ICON
       }),
-      div({ className: "page-heading-text " + this.props.iconSize}, [
-        h2({ id: this.props.id + "_title", className: "page-heading-title  " + this.props.color + "-color", style: this.props.titleStyle }, [this.props.title]),
-        span({ id: this.props.id + "_description", className: "page-heading-description", style: this.props.descriptionStyle }, [this.props.description]),
+      div(this.props.iconSize === 'none' ?
+        { style: NONE }
+        : this.props.iconSize === 'large' ?
+          { style: LARGE }
+          : { style: MEDIUM }, [
```

Comment:
One last suggestion I have is to move checking what kind of iconSize out to a separate method, and then just applying `{ style: newMethod(this.props.iconSize) }` so it looks cleaner, especially since each style for now is just a different number for the margin size. Otherwise, the changes look great!